### PR TITLE
.html extension removed from contact us page

### LINKED ISSUE #162: Link to contact us page contains .html extension in url
## Description
Link to contact us page http://systers.io/contactus.html This url contains .html extension which is not a good practice. **.html** should be removed from url.

## Mocks
image

## Acceptance Criteria
index.html should be modified accordingly

### Update [Required]
- [ ] index.html

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
5 minutes

### PR DESCRIPTION
**Pull request by Vertika-Shukla**
Contact us page link is updated with no .html extension

### Description
Contact us page which was earlier http://systers.io/contactus.html contained .html extension. This url has been updated and .html extension is removed.

Fixes #162 

### Type of Change:
- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
It has been tested by running it on my browser. 

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
  
**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,160 +1,163 @@
...
-        <li class="toggle"><a href="#main-menu">Contribute<span class="fas fa-chevron-down"></span></a>
-            <ul>
-              <li><a href="/newcomers">Newcomers</a></li>
-              <li><a href="/open-source-workflow">Open Source Workflow</a></li>
-              <li><a href="/member-levels">Member Levels</a></li>
-            </ul>
-        </li>
-            <li class="toggle"><a href="#main-menu">About Us <span class="fas fa-chevron-down"></span></a>
-            <ul>
-              <li><a href="/contactus.html">Contact Us</a></li>
```
Comment (sara-02) [inline index.html]: @Vertika-Shukla Looks like this was the only change that was intended. But your text editior auto-modified the whole file. Not a good practise.

File: index.html [hunk 2]
```diff
@@ -1,160 +1,163 @@
...
+    <div class="header"><h1>Welcome to Systers' Open Source!</h1></div>
+        <p>Systers is an international <a href="https://anitab.org/" target="_blank">Anita B.org</a> community encouraging the participation of all women involved in technology.</p>    
+    
+        <p>We are committed to providing a friendly, safe, and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, work experience, or religion (or lack thereof).</p>
+
+    <div class="subheader" id="join-systers"><h2>Join Systers</h2></div>
+        <p> In order to engage with the community, you can sign up on <a href="http://systers.io/slack-systers-opensource/" target="_blank">Systers Open Source Slack</a> and introduce yourself in the #intro channel, then subscribe to our <a href="http://systers.org/mailman/listinfo/systers-opensource" target="_blank">systers-dev mailing list</a> for more opportunities.</p>
+
+        <p>You can also attend the Systers Weekly Update meetings every Thursday at 12:00 Eastern Standard Time (17:00 UTC) on Zoom.  You'll find the meeting details on our <a href="/contactus.html">contact page</a>.</p>
+
```
Comment (sara-02) [inline index.html]: @Vertika-Shukla Looks like you missed removing `.html` from here.

Comment (Tharangi) [reply index.html]: I think in @Vertika-Shukla's issue she has mentioned on fixing on the main navigation only. We can create a separate issue for this issue too. Anyways, @Vertika-Shukla are you fixing the line @sara-02 has mentioned here?

**Comment by Tharangi** [comment]
@divyanshu-rawat What do you think about this PR?

**Comment by divyanshu-rawat** [comment]
@Tharangi this PR has merge conflicts and even @Vertika-Shukla is not responding.
I don't know what's going on in here.
what do you think?

**Comment by ancyp** [comment]
@divyanshu-rawat we are doing this in GSoC. So close this?

**Comment by BaaniLeen** [comment]
Taken care of by in the new Contact Us Page - PR #188

**Pull request closed without merging**